### ISSUE TITLE
SURVIVOR not reporting DR

### ISSUE THREAD

**Issue by Akazhiel** [fritzsedlazeck/SURVIVOR#146]
Hello!

Upon using SURVIVOR to merge 2 VCF files from the same caller, I've noticed that the `DR` field always report `0,0` despite the reads information being available in both the VCF files. What could be the reason of this?

Best regards,
Jonatan

**Comment by fritzsedlazeck** [comment]
Which SV caller are you using ? 
VCF files are always different between callers and that makes it sometimes hard to parse this field correctly

**Comment by Akazhiel** [comment]
I am using different variant callers since I'm trying to call for somatic mutations and thus I need to verify the variants are called in different callers. For this exact issue I was just merging 2 VCF from the same caller, one for Normal and one for Tumor. The variant caller used was SVIM.

**Comment by johannesgeibel** [comment]
Hi,
I actually also run into the same problem merging sniffles vcf output files with the latest commit of SURVIVOR. The DR field is correctly copied into the position for the reference allele in the new vcf, while the info of the DV field is removed. A fix for some common callers would be a nice enhancement.
